XiEditor/EditViewController.swift

```diff
@@ -127,6 +127,13 @@ class EditViewController: NSViewController, EditViewDataSource, FindDelegate, Sc
             updatePluginMenu()
         }
     }
+    
+    /// Current language used for syntax highlighting
+    var currentLanguage: String? {
```

Comment:
I might need to think this through a bit more, but I think we can safely just init this to "Plain Text". xi-mac, as a product, will always includes a bundled syntax highlighting plugin, which should always support this syntax?

Reply:
I think this is fine for now, although not fine in the future. What if we change how language identifiers are spelt, for instance? Like "text/plain" instead of "Plain Text" or something.

I think this logic (default language) belongs to the core, but currently, it's hard to implement because of the way we're loading languages there.

Reply:
There's also an issue that we're sending empty language from the core down the wire (`language_changed {"view_id": "view-id-1", "language_id": ""}`), so whatever default we provide will be overwritten, and checking this in Swift code feels ugly. Maybe we should tackle this in a separate issue? Rethink how the core is loading languages?

Reply:
yep that sounds reasonable.